src/commands/instances/create.ts

```diff
@@ -12,9 +12,14 @@ export default class CreateCommand extends Command {
     }),
     integration: Flags.string({
       char: "i",
-      required: true,
+      required: false,
       description: "ID of the integration this is an instance of",
     }),
+    version: Flags.string({
+      char: "i",
```

Comment:
Should this be `"v"`? Won't it conflict with the argument for `integration`?

Reply:
Oops. oversight on my part. I just noticed that config-vars also currently has shorthand "v" That would also cause a conflict, what character should I use? And should the short hand also be updated in the "update" file?

Reply:
Hmm, good point. I might advocate to just not have this argument then, and instead change the description of the existing `integration` argument to indicate that it can be either an integration id OR a specific integration version id.